### DISCUSSION TITLE
Maximum UDP payload without fragmentation?

### DISCUSSION THREAD

**Discussion by LuisFGomez** [openthread/openthread#8691] [Q&A] [answered]
Hello all! Does anybody know what the maximum payload in a UDP datagram can be without fragmentation in OpenThread? Thanks!

**Comment by jwhui** [comment, marked as answer]
The maximum UDP payload depends on a number of things. 6LoWPAN header compression depends on what values are used in the IPv6, 6lowpan, and IEEE 802.15.4 headers. If the packet is traversing multiple mesh hops, the IPv6 Mesh Addressing header needs to be included. As for a ballpark number, I would say around 60-70 bytes.

One way to learn more is by observing a packet trace using Wireshark.
